acceptance/lib/helper.rb

```diff
@@ -196,6 +196,22 @@ def get_defaults_var(host, varname)
     stdout
   end
 
+  def hup(host = master, timeout = 30)
+    pid = on(host, "cat /var/run/puppetlabs/puppetserver/puppetserver").stdout
+    on(host, "kill -HUP #{pid}")
+    url = "https://#{host}:8140/puppet/v3/status"
+    curlcmd = "curl -s -k -X GET #{url}"
```

Comment:
This is so awful and wrong.  Gnashing of teeth.  Use http_request.  We just need any response code.